qt/widgets/common/CMakeLists.txt

```diff
@@ -734,67 +734,71 @@ if ( ENABLE_WORKBENCH )
   find_package ( QScintillaQt5 REQUIRED )
 endif()
 
-mtd_add_qt_library (TARGET_NAME MantidQtWidgetsCommon
-  QT_VERSION 4
-  SRC ${SRC_FILES}
-  MOC ${MOC_FILES}
-  NOMOC ${INC_FILES}
-  UI ${UI_FILES}
-  DEFS
-    IN_MANTIDQT_COMMON
-    QSCINTILLA_DLL
-  QT4_DEFS
-    USE_QTWEBKIT
-  INCLUDE_DIRS
-    inc
-  LINK_LIBS
-    ${TARGET_LIBRARIES}
-  QT4_LINK_LIBS
-    Qt4::QtHelp
-    Qt4::QtNetwork
-    Qt4::QtWebKit
-    Qt4::Qscintilla
-  INSTALL_DIR
-    ${LIB_DIR}
-  OSX_INSTALL_RPATH
-    @loader_path/../MacOS
-    @loader_path/../Libraries
-  LINUX_INSTALL_RPATH
-    "\$ORIGIN/../${LIB_DIR}"
-)
+if ( ENABLE_MANTIDPLOT )
```

Comment:
Are these `if` blocks necessary. I thought `mtd_add_qt_library` was setup to skip Qt4  targets if `ENABLE_MANTIDPLOT=FALSE` [QtTargetFunctions.cmake](https://github.com/mantidproject/mantid/blob/master/buildconfig/CMake/QtTargetFunctions.cmake#L23)? The idea was to reduce the number of places we would need this check.

Reply:
Oh, good call, didn't catch that. Removing and testing now.